<launch> 
    <node pkg="assignment2" type="apriltag_locator" name="apriltag_locator" output="log"/>
    <node pkg="assignment2" type="node_A" name="node_A" output="screen"/>
</launch>
